tests/integration/v2_api/test_override_requests.py

```diff
@@ -25,3 +25,36 @@ def foo(self, docs, **kwargs):
 
     assert resp1[0].docs[0].text == ''
     assert resp2[0].docs[0].text == 'foo called'
+
+
+def test_override_requests_uses_after():
+    class FooExecutor(Executor):
+        @requests(on='/bar')
+        def foo(self, docs, **kwargs):
+            for doc in docs:
+                doc.text = 'foo called'
+
+    class OtherExecutor(Executor):
+        @requests(on='/bar')
+        def bar(self, docs, **kwargs):
+            for doc in docs:
+                doc.text = 'bar called'
+
+    with Flow(port_expose=exposed_port).add(
+        uses=FooExecutor,
+        uses_requests={'/foo': 'foo'},
```

Comment:
Is the `bar` available on `/bar`? Can you add a test to assert this?

Reply:
No, all good for this PR, from what I can tell. Just wondering in terms of general usage. I would expect that OtherExecutor still has its endpoint bound to /bar

Reply:
I agree, might be worth a change

Reply:
Re-read your comment, understood it this time, and added a test accordingly ;)

Reply:
Oh so it does preserve the OtherExecutor.bar binding. Good